spec/formatting.md

```diff
@@ -219,15 +229,24 @@ the following steps are taken:
    as long as reasonable precautions are taken to keep the function interface
    simple and minimal, and avoid introducing potential security vulnerabilities.
 
-   As implementations MAY allow custom functions to be defined by users,
-   their access to the _formatting context_ SHOULD be minimal and read-only,
-   and their execution time SHOULD be limited.
+   An implementation MAY define its own functions.
+   An implementation MAY allow custom functions to be defined by users.
 
```

Comment:
This too is out of place, and should be stated elsewhere.

```suggestion
```

Reply:
Where?